src/components/level.js

```diff
@@ -21,7 +26,13 @@ function PureLevelItem({statistic, total, delta}) {
 
   return (
     <React.Fragment>
-      <h5>{t(capitalize(statistic))}</h5>
+      <h5>
+        {t(capitalize(statistic)) +
+          ' ' +
+          (statistic !== 'confirmed'
+            ? getPercentageText(confirmed, total)
+            : '')}
```

Comment:
@jagadeshanh  don't need to write ternary operator for this
`(statistic !== 'confirmed' && getPercentageText(confirmed, total))`

Reply:
Thank you I will update the PR.